sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/GatewayConnectionConfig.java

```diff
@@ -43,6 +45,13 @@ public static GatewayConnectionConfig getDefaultConfig() {
         return new GatewayConnectionConfig();
     }
 
+    public boolean getThinClientEnabled() { return this.thinclientEnabled; }
```

Comment:
This should not be public API - just environment variable. Eventually we will always enable ThincClient (when service has enabled it for account)in Gateway mode. Initially we will make it opt-in during bake-time - but env variable/system-property is sufficient.